.github/workflows/solidity-foundry.yml

```diff
@@ -18,10 +18,13 @@ jobs:
         with:
           filters: |
             src:
-              - 'contracts/**/*'
+              - 'contracts/src/v0.8/**/*'
```

Comment:
Foundry only exists in v0.8, maybe we can make this even more exact after cleaning up the folder